play-testkit/src/test/java/akka/grpc/play/test/PlayJavaFunctionalTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.play.test;
+
+import akka.grpc.play.AkkaGrpcClientHelpers;
+import akka.grpc.play.AkkaGrpcClientHelpers$;
+
+import example.myapp.helloworld.grpc.helloworld.*;
+
+import org.junit.*;
+
+import play.api.test.TestServerFactory;
+import scala.concurrent.Future;
+import scala.reflect.ClassTag$;
+
+import scala.compat.java8.FutureConverters$;
+
+import play.*;
+import play.api.routing.*;
+import play.api.test.DefaultTestServerFactory$;
+import play.api.test.NewTestServer;
+import play.inject.guice.*;
+import play.libs.ws.*;
+import play.test.*;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.*;
+import static play.inject.Bindings.*;
+
+public final class PlayJavaFunctionalTest implements AkkaGrpcClientHelpers {
+  private static final GreeterService$ GreeterService = GreeterService$.MODULE$;
```

Comment:
isn't it worrying that we need to access `MODULE$` in a java test here?
Is this using the scala or java generated things?

Seems a bit weird for the Service hm...

Reply:
I guess it's using Scala generated things? I'll look into this.